vars/buildWDKAutoSwitch.groovy

```diff
@@ -153,14 +165,15 @@ def call(Map config = [unityVersions:[]]) {
                     dir (directoryName) {
                       checkout scm
                       unstash 'setup_w'
-                      // gradleWrapper "--refresh-dependencies"
-                      gradleWrapper "-Prelease.stage=${params.RELEASE_TYPE.trim()} -Prelease.scope=${params.RELEASE_SCOPE} check --info"
+
+                      def logLevel = params.LOG_LEVEL ? config.logLevel : params.LOG_LEVEL
+                      gradleWrapper "-Prelease.stage=${params.RELEASE_TYPE.trim()} -Prelease.scope=${params.RELEASE_SCOPE} check --${logLevel.toLowerCase()}"
```

Comment:
ideally each invocation of gradlew should pass the loglevel. But I think it should be doable to fetch the paramters from inside the `gradleWrapper` task so that we have no need to pass this down explicitly.